__tests__/inlineStyles.test.js

```diff
@@ -0,0 +1,234 @@
+import React from 'react';
+import { View, Button } from 'react-native';
+import { render, fireEvent, waitFor } from '@testing-library/react-native';
+import Animated, {
+  useSharedValue,
+  withTiming,
+  withSequence,
+  withSpring,
+  withRepeat,
+  withDelay,
+} from '../src';
+
+const AnimatedSharedValueComponent = (props) => {
+  const widthSV = props.sharedValue;
+
+  return (
+    <View>
+      <Animated.View testID="view" style={props.style} />
+      <Button
+        testID="button"
+        title="toggle"
+        onPress={() => {
+          widthSV.value = 200;
+        }}
+      />
+    </View>
+  );
+};
+
+const AnimatedComponent = ({ style }) => {
+  const sharedVal = useSharedValue(100);
+  return (
+    <AnimatedSharedValueComponent
+      sharedValue={sharedVal}
+      style={style(sharedVal)}
+    />
+  );
+};
+
+describe('inline styles', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.runOnlyPendingTimers();
+    jest.useRealTimers();
+  });
+
+  it('works with shared value', async () => {
+    jest.useRealTimers();
+    const { getByTestId } = render(
+      <AnimatedComponent style={(sharedVal) => ({ width: sharedVal })} />
+    );
+    const view = getByTestId('view');
+    const button = getByTestId('button');
+
+    expect(view).toHaveAnimatedStyle({ width: 100 });
+
+    fireEvent.press(button);
+    await waitFor(() => {
+      expect(view).toHaveAnimatedStyle({ width: 200 });
+    });
+    jest.useFakeTimers();
```

Comment:
Yes, because in `afterEach` callback we call `jest.runOnlyPendingTimers();` and it's going to show a long annoying warning. And `afterEach` is needed for all other tests that rely on fake timers. And this test needs real timers.